site/about/glossary/_mrm.qmd

```diff
@@ -0,0 +1,44 @@
+#### Model risk management
+
+1st line of defense
+: Business unit(s) responsible for model development, initial validation, and implementation during the model lifecycle. As the 1st line of defense, model developers must document and test models to ensure that they are accurate, robust, and fit for purpose.
+
+2nd line of defense
+: An independent oversight function that provides a governance framework for the model lifecycle. As the 2nd line of defense, model validators must independently validate and challenge models created by model developers to ensure that model risk management principles are followed. 
+
+3rd line of defense
+: Typically an internal audit function responsible for providing an independent and comprehensive review of the risk management processes and controls that the first two lines have implemented. 
+
+model developer
+: Responsible for the design, implementation, and maintenance of models to ensure they are fit-for-purpose, accurate, and aligned with business requirements. As subject matter experts, they collaborate with model validators and other business units, ensuring the models are conceptually sound and robust. 
+
+model governance
+: A framework of policies, procedures, and standards established to oversee the lifecycle of models within an organization. Ensures that models are developed, validated, implemented, and retired in a controlled and consistent manner, promoting accountability, transparency, and adherence to regulatory requirements.
+
+model implementation
+: A collaborative effort among model developers, model owners, and model users. Model implementation includes a formalized implementation plan and associated procedures, a review of results, and a record of model change procedures. 
```

Comment:
```suggestion
: A collaborative effort among model developers and model owners. Model implementation includes a formalized implementation plan and associated procedures, a review of results, and a record of model change procedures. 
```

Not sure how often model users would be involved in the model development but I think it's perhaps less common than suggested here. E.g. a mortgage specialist at a large financial institution would be a model user but likely would not be involved in the development of the model.

Reply:
This is something that is brought up in the Intro to MRM book, but I also can't attest to its accuracy!